### ISSUE TITLE
New libraries

### ISSUE THREAD

**Issue by axel-klinger** [axel-klinger/BibsOnGitHub#2]
Please comment if you know more libraries on GitHub or GitLab with

Country, City, Name, URL

**Comment by zuphilip** [comment]
Why not continue to collect all data in the main repository at https://github.com/hbunke/BibsOnGitHub instead of doing this in this fork here?

**Comment by axel-klinger** [comment]
Yes, I would like to sync it with the original list, but I'm also looking for libraries outside of europe. Beside this it would be nice to have the main list sorted, for example by country and city (if it gets this attributes), to make it easier to sync.

**Comment by GJagusch** [comment]
https://github.com/ULB-Darmstadt ULB Darmstadt on GitHub

**Comment by axel-klinger** [comment]
ULB is now included!
